examples/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh 
+
+if [ ! -d message-schemas ]; then
+    git clone https://github.com/foxglove/message-schemas
+fi
+
+if [ ! -d ros ]; then
+    find message-schemas/proto/ros -name \*.proto -exec protoc -I message-schemas/proto {} --python_out . \;
```

Comment:
It might be better to delete the `message-schemas/proto/ros` directory unconditionally, git pull, and run this each setup `setup.sh` is run, to account for new or deleted proto files.

Reply:
Good point I'll make that change in an update.